python/ray/tune/execution/tune_controller.py

```diff
@@ -253,6 +253,20 @@ def __init__(
 
             self._legacy_remote_experiment_path = remote_experiment_path
 
+            if (
+                self._legacy_local_experiment_path
+                and self._legacy_remote_experiment_path
+                and Path(self._legacy_local_experiment_path)
+                == Path(self._legacy_remote_experiment_path)
+            ):
+                warnings.warn(
+                    "The local experiment path is the same as the remote "
+                    "experiment path. Set a different `storage_path` or raise an "
+                    "issue on GitHub if this issue persists. Deactivating the"
+                    "remote experiment path."
+                )
+                self._legacy_remote_experiment_path = None
+
```

Comment:
In Windows, we resolved the default storage path to `C:/Users/username/ray_results`. When passing the path on, it was interpreted as a remote path in downstream objects (because we used urlparse to check for local/non-local paths and that doesn't work with windows paths).

So we end up in a situation where the remote path is `C:/Users/username/ray_results`, and the local path gets resolved _again_ to the same path, `C:/Users/username/ray_results`. We write to the path locally and the syncer tries to sync up to that path. This results in various different error messages.